atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/TransactionAborter.java

```diff
@@ -132,27 +124,25 @@ private static boolean isInRange(
                 "Found transaction to abort",
                 SafeArg.of("startTimestamp", startTimestamp),
                 SafeArg.of("commitTimestamp", commitTimestamp),
-                SafeArg.of("keyspace", keyspaceName),
+                SafeArg.of("keyspace", namespace),
```

Comment:
Left the safearg as keyspace since that's what it's called in config. Can rename to namespace to be consistent with arg (and I think new term?)